index.html

```diff
@@ -26,6 +26,12 @@
 				<div class="bar"></div>
 				<div class="bar"></div>
 				<div class="bar"></div>
+				<ul class="nav-list">
+					<li class="nav-item"><a href="">Portfolio</a></li>
+					<li class="nav-item"><a href="">About</a></li>
+					<li class="nav-item"><a href="">Contact</a></li>
+					<li class="nav-item"><a href=""><img src="media/mail-icon.png" alt="Mail Icon" class="mail-icon"></a></li>
+				</ul>
```

Comment:
- [ ] Nice work implementing the navigation menu in the desktop version, however, the email icon is not currently being displayed correctly as shown below 🔽, please make sure to include the icon in your repo as it is not existent right now

### Figma Design
![Figma nav](https://user-images.githubusercontent.com/104319462/220783605-1454f8a7-dafd-492f-a206-961174f0d62d.png)
### Your Design
![user design nav](https://user-images.githubusercontent.com/104319462/220783643-990ed3d9-3c91-4e90-99ff-b32b81616400.png)